src/vulkan/vulkan_presenter.h

```diff
@@ -183,6 +183,17 @@ namespace dxvk::vk {
       return m_swapchain;
     }
 
+    /**
+     * \brief Gets list of supported surface formats
+     *
+     * \param [out] formats List of supported surface formats
+     * \param [in] fullScreenExclusive Fullscreen exclusive state
+     * \returns Status of the operation
+     */
+    VkResult getSupportedFormats(
```

Comment:
Can we expose a `supportsColorSpace` kind of function instead? Having to pass somewhat unrelated data around and also receiving info that we don't need for that purpose (the exact VkFormat) is kind of bad design.

Reply:
Sure

Reply:
Done.